### DISCUSSION TITLE
I need your help

### DISCUSSION THREAD

**Discussion by lzb-libra** [dbeaver/dbeaver#12988] [Q&A]
When I set dbeaver recently, I don't know what causes me not to display the result set when I execute multiple SQL statements at the same time。When I execute a SQL statement, the result set is normally displayed, but when I execute multiple SQL statements at the same time, I only have this page
![image](https://user-images.githubusercontent.com/63389341/123391645-d052d200-d5ce-11eb-95c8-992606b71a4b.png)
What settings should I modify to execute multiple SQL, and multiple SQL will display the result set

**Comment by ShadelessFox** [comment]
Hello @lzb-libra,

Your database may not support that feature.

Instead, you can use `Execute SQL Script` (<kbd>Alt+X</kbd>) to execute multiple SQL scripts.

More information you can find on our wiki: https://github.com/dbeaver/dbeaver/wiki/SQL-Execution

**Reply by lzb-libra** [reply]
I am sure that my database is supported. At the beginning, I was able to execute it like this. Now the dbeaver installed on my other computer can meet my expectations. That is, one day I set up something and then I can't

**Reply by ShadelessFox** [reply]
You can try resetting your preferences using <kbd>File</kbd> &rArr; <kbd>Reset workspace state</kbd>, but I doubt there's something wrong with them.

Are you sure your statements produce results?

**Reply by lzb-libra** [reply]
I tried to say "reset workspace state" and didn't achieve the effect I wanted
The first picture is the result set of multiple statements I executed on windows, and the second picture is the result set of multiple SQL statements I executed on another computer。
![image](https://user-images.githubusercontent.com/63389341/123396136-8c160080-d5d3-11eb-940e-903120ba8643.png)
![image](https://user-images.githubusercontent.com/63389341/123396570-0777b200-d5d4-11eb-91a4-12405c22747b.png)

**Reply by ShadelessFox** [reply]
Can you record a video showing our SQL script and the way you execute it?

**Reply by lzb-libra** [reply]
I don't know how to send you a video

**Reply by ShadelessFox** [reply]
You can upload it here, GitHub allows attachments up to 100MB.

**Reply by lzb-libra** [reply]
I tried to upload the video, but it prompted me "This video is too big. Try again with a file size less than 10MB"

**Reply by ShadelessFox** [reply]
I see you've attached screenshots containing SQL, but what database is it and what driver are you using?

**Reply by lzb-libra** [reply]
Thank you very much for your help. I have compared the settings with another computer one by one. After changing the settings to another computer, I have achieved the results I want

**Reply by ShadelessFox** [reply]
I'm glad your issue was solved. Can you share which settings you have tuned to help others who will encounter the same problem?
